tap-gui/plugins/application-accelerator-git-repo.hbs.md

```diff
@@ -45,6 +45,31 @@ These steps describe an example configuration that uses GitHub:
    For more information, see the
    [Backstage documentation](https://backstage.io/docs/integrations/github/locations).
 
+### Using kubernetes secrets
```

Comment:
"kubernetes" should be capitalized

Reply:
I'll fix this and any other writing errors I spot